govcd/external_network_v2.go

```diff
@@ -0,0 +1,189 @@
+/*
+ * Copyright 2020 VMware, Inc.  All rights reserved.  Licensed under the Apache v2 License.
+ */
+
+package govcd
+
+import (
+	"fmt"
+	"net/url"
+
+	"github.com/vmware/go-vcloud-director/v2/types/v56"
+)
+
+// ExternalNetworkV2 is a type for version 2 of external network which uses OpenAPI endpoint to manage external networks
+type ExternalNetworkV2 struct {
+	ExternalNetwork *types.ExternalNetworkV2
+	client          *Client
+}
+
+// CreateExternalNetwork creates a new external network using OpenAPI endpoint
+func CreateExternalNetworkV2(vcdClient *VCDClient, newExtNet *types.ExternalNetworkV2) (*ExternalNetworkV2, error) {
```

Comment:
why we don't make vcdClient the `recipient` ?

Reply:
Can do - the only reason is I tried to follow consistency. We already have `func CreateExternalNetwork(vcdClient *VCDClient, externalNetworkData *types.ExternalNetwork) (Task, error) {`

Reply:
@lvirbalas @dataclouder

Reply:
No strong preference here, but like consistency :)